index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="flex-container">
+      <header>
+        <a href="" class="white text-normal align-center roboto">Mudasir Sherwani</a>
+        <nav>
+            <i class="fa-solid fa-bars fa-2x white"></i>
+        </nav>
+      </header>
+      <section>
+        <div class="shining-star"></div>
+        <div class="developer-title align-center" >
+          <h1 class="developer-name">Mr. Mudasir</h1>
+          <p class="software-developer">Software Developer</p>
+        </div>
```

Comment:
- [ ] The subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)`. Kindly implement it as it is a subtitle.